modules/migration-using-must-gather.adoc

```diff
@@ -23,29 +32,70 @@ You can collect data for a one-hour or a 24-hour period and view the data with t
 .Procedure
 
 . Navigate to the directory where you want to store the `must-gather` data.
-. Run the `oc adm must-gather` command:
-
-* To gather data for the past hour:
+. Run the `oc adm must-gather` command for one of the following options:
+
+ifdef::troubleshooting-3-4,troubleshooting-mtc[]
+* To collect data for the past hour:
+endif::[]
+ifdef::oadp-troubleshooting[]
+* Full `must-gather`, including Prometheus metrics:
+endif::[]
 +
 [source,terminal,subs="attributes+"]
 ----
-$ oc adm must-gather --image=registry.redhat.io/rhmtc/openshift-migration-must-gather-rhel8:v{mtc-version}
+$ oc adm must-gather --image={must-gather}
 ----
 +
-The data is saved as `/must-gather/must-gather.tar.gz`. You can upload this file to a support case on the link:https://access.redhat.com/[Red Hat Customer Portal].
+The data is saved as `must-gather/must-gather.tar.gz`. You can upload this file to a support case on the link:https://access.redhat.com/[Red Hat Customer Portal].
 
-* To gather data for the past 24 hours:
+ifdef::oadp-troubleshooting[]
+* Essential `must-gather`, without Prometheus metrics, for a specific time duration:
```

Comment:
after instances of `must-gather` in this list, I think there should be "data", so here Essential `must-gather` data, without Prometheus metrics....